packages/react-native/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -545,7 +552,15 @@ - (void)focus
 
 - (void)blur
 {
+#if !TARGET_OS_OSX // [macOS]
   [_backedTextInputView resignFirstResponder];
+#else
+  NSWindow *window = _backedTextInputView.window;
+  if (window && window.firstResponder == _backedTextInputView) {
```

Comment:
Same as above. `window.firstResponder` should return `nil` if `window` is `nil`:
```suggestion
  if (window.firstResponder == _backedTextInputView) {
```